### DISCUSSION TITLE
分享一个有趣的项目

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#104] [1️⃣ 新鲜事]
项目地址：https://github.com/CollegesChat/university-information
该项目主要是统计了一些国内大学的生活水平状况，统计方式是问卷的形式。
有趣的点在于，该项目将统计的问卷结果通过ci/cd的方式生成到仓库中一个新分支上，以开源的方式做问卷，并将结果也开源出来。
统计结果网址：https://github.com/CollegesChat/university-information/tree/generated

**Comment by will-ww** [comment]
挺有意思的~
